sql/01-create-table-student.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE STUDENT()
```

Comment:
whats happening with this table here? is this the first excercise for 4? 
also can you name your branches by features
so for example - ```day-3-excercises vs. workingbranch```